x-pack/legacy/plugins/code/public/components/shared/icons.tsx

```diff
@@ -58,6 +58,17 @@ export const GoIcon = () => (
   </svg>
 );
 
+export const CtagsIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
+    <rect width="32" height="32" fill="white" />
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M9.38147 22.4545L1.49428 14.5673C1.02148 14.0945 0.899259 13.8738 0.818851 13.6085C0.738443 13.3433 0.738443 13.072 0.818851 12.8067C0.899259 12.5414 1.02148 12.3207 1.49428 11.8479L11.1446 2.19753C11.4185 1.92366 11.5232 1.83884 11.6524 1.75804C11.7816 1.67724 11.913 1.62282 12.0615 1.58859C12.21 1.55437 12.3439 1.54034 12.7312 1.54034H20.4903C21.1579 1.54034 21.3999 1.60967 21.6441 1.73994C21.8883 1.87022 22.0801 2.06144 22.2111 2.30524C22.342 2.54903 22.4121 2.79091 22.414 3.45844L22.4365 11.1838C22.4376 11.5721 22.4239 11.7065 22.3898 11.8556C22.3558 12.0046 22.3014 12.1365 22.2205 12.2662C22.1395 12.3958 22.0545 12.5009 21.7799 12.7754L12.1009 22.4545C11.6281 22.9273 11.4074 23.0495 11.1421 23.1299C10.8768 23.2103 10.6055 23.2103 10.3402 23.1299C10.0749 23.0495 9.85427 22.9273 9.38147 22.4545ZM11.0402 21.3938L20.7193 11.7148C20.8576 11.5765 20.9085 11.5228 20.9317 11.4943C20.9353 11.4577 20.9371 11.3837 20.9365 11.1882L20.914 3.4628C20.9132 3.1916 20.9096 3.09541 20.9017 3.05144C20.8577 3.04366 20.7615 3.04035 20.4903 3.04035H12.7312C12.5359 3.04035 12.4621 3.0423 12.4255 3.04601C12.397 3.06924 12.3434 3.12009 12.2053 3.2582L2.55494 12.9086C2.36296 13.1005 2.29722 13.171 2.2716 13.2076C2.29722 13.2442 2.36296 13.3147 2.55494 13.5067L10.4421 21.3938C10.6341 21.5858 10.7045 21.6516 10.7412 21.6772C10.7778 21.6516 10.8482 21.5858 11.0402 21.3938ZM17.4379 6.54402C16.8521 5.95824 15.9023 5.95824 15.3166 6.54402C14.7308 7.12981 14.7308 8.07956 15.3166 8.66534C15.9023 9.25113 16.8521 9.25113 17.4379 8.66534C18.0237 8.07956 18.0237 7.12981 17.4379 6.54402ZM10.1443 16.8534L7.10054 13.8096C7.07478 13.7898 7.05001 13.7681 7.02642 13.7445C6.81172 13.5298 6.7544 13.2173 6.85448 12.9504C6.89078 12.8508 6.94891 12.7572 7.02886 12.6773C7.04481 12.6613 7.0613 12.6462 7.07828 12.632L10.6477 9.06256C10.9406 8.76967 11.4155 8.76967 11.7084 9.06256C12.0013 9.35546 12.0013 9.83033 11.7084 10.1232L8.62194 13.2097L10.7433 15.331L13.8299 12.2444C14.1228 11.9516 14.5976 11.9516 14.8905 12.2444C15.1834 12.5373 15.1834 13.0122 14.8905 13.3051L11.2692 16.9264C10.9763 17.2193 10.5014 17.2193 10.2085 16.9264C10.1853 16.9032 10.1639 16.8787 10.1443 16.8534Z"
+    />
+  </svg>
+);
+
 export const BinaryFileIcon = () => (
```

Comment:
@Poytr1 I think the issue here is that the icon doesn't match the one I uploaded to the issue. It was set originally to 24px and will need to be resized using CSS rather than changing the dimensions of the SVG `rect` or `viewBox`. Below is the raw SVG code from the file provided in https://github.com/elastic/code/issues/1361

```
<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M9.38147 22.4545L1.49428 14.5673C1.02148 14.0945 0.899259 13.8738 0.818851 13.6085C0.738443 13.3433 0.738443 13.072 0.818851 12.8067C0.899259 12.5414 1.02148 12.3207 1.49428 11.8479L11.1446 2.19753C11.4185 1.92366 11.5232 1.83884 11.6524 1.75804C11.7816 1.67724 11.913 1.62282 12.0615 1.58859C12.21 1.55437 12.3439 1.54034 12.7312 1.54034H20.4903C21.1579 1.54034 21.3999 1.60967 21.6441 1.73994C21.8883 1.87022 22.0801 2.06144 22.2111 2.30524C22.342 2.54903 22.4121 2.79091 22.414 3.45844L22.4365 11.1838C22.4376 11.5721 22.4239 11.7065 22.3898 11.8556C22.3558 12.0046 22.3014 12.1365 22.2205 12.2662C22.1395 12.3958 22.0545 12.5009 21.7799 12.7754L12.1009 22.4545C11.6281 22.9273 11.4074 23.0495 11.1421 23.1299C10.8768 23.2103 10.6055 23.2103 10.3402 23.1299C10.0749 23.0495 9.85427 22.9273 9.38147 22.4545ZM11.0402 21.3938L20.7193 11.7148C20.8576 11.5765 20.9085 11.5228 20.9317 11.4943C20.9353 11.4577 20.9371 11.3837 20.9365 11.1882L20.914 3.4628C20.9132 3.1916 20.9096 3.09541 20.9017 3.05144C20.8577 3.04366 20.7615 3.04035 20.4903 3.04035H12.7312C12.5359 3.04035 12.4621 3.0423 12.4255 3.04601C12.397 3.06924 12.3434 3.12009 12.2053 3.2582L2.55494 12.9086C2.36296 13.1005 2.29722 13.171 2.2716 13.2076C2.29722 13.2442 2.36296 13.3147 2.55494 13.5067L10.4421 21.3938C10.6341 21.5858 10.7045 21.6516 10.7412 21.6772C10.7778 21.6516 10.8482 21.5858 11.0402 21.3938ZM17.4379 6.54402C16.8521 5.95824 15.9023 5.95824 15.3166 6.54402C14.7308 7.12981 14.7308 8.07956 15.3166 8.66534C15.9023 9.25113 16.8521 9.25113 17.4379 8.66534C18.0237 8.07956 18.0237 7.12981 17.4379 6.54402ZM10.1443 16.8534L7.10054 13.8096C7.07478 13.7898 7.05001 13.7681 7.02642 13.7445C6.81172 13.5298 6.7544 13.2173 6.85448 12.9504C6.89078 12.8508 6.94891 12.7572 7.02886 12.6773C7.04481 12.6613 7.0613 12.6462 7.07828 12.632L10.6477 9.06256C10.9406 8.76967 11.4155 8.76967 11.7084 9.06256C12.0013 9.35546 12.0013 9.83033 11.7084 10.1232L8.62194 13.2097L10.7433 15.331L13.8299 12.2444C14.1228 11.9516 14.5976 11.9516 14.8905 12.2444C15.1834 12.5373 15.1834 13.0122 14.8905 13.3051L11.2692 16.9264C10.9763 17.2193 10.5014 17.2193 10.2085 16.9264C10.1853 16.9032 10.1639 16.8787 10.1443 16.8534Z"/>
</svg>
```

### A better explanation

SVGs use a `viewBox` as a type of canvas. All the floats inside the `path` object relate to an absolute position and tell the browser how to draw the path. Modifying the size of the `viewBox` doesn't actually scale the SVG — instead it only resizes the 'canvas' so-to-speak. With this icon as an example — the icons path lengths will always be relative to the top left of the `viewBox`.

Reply:
Thanks for all the details! Will update this PR soon.

Reply:
Just change width and height to 32, and keep `viewBox= "0 0 24 24"`, looks good now 😃
![image](https://user-images.githubusercontent.com/20534718/60637335-a4237100-9e4c-11e9-94e1-4393c3730475.png)